starter-code/index.js

```diff
@@ -1,10 +1,32 @@
-var SortedList = function() {};
+var SortedList = function(items, length) {
+  this.items = [];
+  this.length = 0;
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+SortedList.prototype.add = function(item) {
+  this.items.push(item);
+  this.items.sort((a, b) => {
+    return a - b;
+  });
+  this.length++;
+};
+SortedList.prototype.get = function(pos) {
```

Comment:
what if you ask for position of an element on position 6 and you have less elements? you have let your user know that the element on that position doesn't exist. so before this return you would put something like this:
```
if (pos < 1 || pos > this.length){
        throw Error("OutOfBounds");
    }
```